sdk/keyvault/ci.yml

```diff
@@ -20,7 +24,11 @@ pr:
       - feature/v4
   paths:
     include:
-      - sdk/keyvault/
+      - sdk/keyvault/ci.yml
+      - sdk/keyvault/keyvault-certificates/
```

Comment:
@praveenkuttappan should we use an exclude model here? The only reason I ask is because is because we likely also want this pipeline to trigger for other files in the root of the keyvault directory like the arm template for tests.